src/driver.toit

```diff
@@ -10,6 +10,11 @@ RESOLUTION_14_BITS ::= 0b01
 RESOLUTION_16_BITS ::= 0b10
 RESOLUTION_18_BITS ::= 0b11
 
+GAIN_AMPLIFIER_1 ::= 0b00
+GAIN_AMPLIFIER_2 ::= 0b01
+GAIN_AMPLIFIER_4 ::= 0b10
+GAIN_AMPLIFIER_8 ::= 0b11
```

Comment:
Feels like there isn't much win by using bits. (same is true for the resolution).
But I'm ok either way.

```suggestion
GAIN_AMPLIFIER_1 ::= 0
GAIN_AMPLIFIER_2 ::= 1
GAIN_AMPLIFIER_4 ::= 2
GAIN_AMPLIFIER_8 ::= 3
```